L1Trigger/TrackFindingTracklet/interface/PurgeDuplicate.h

```diff
@@ -37,6 +37,12 @@ namespace trklet {
 
   private:
     double getPhiRes(Tracklet* curTracklet, const Stub* curStub);
+    bool isSeedingStub(int, int, int);
+    std::string l1tinfo(const L1TStub*, std::string);
+    std::pair<int, int> findLayerDisk(const Stub*);
+    std::vector<double> getInventedCoords(unsigned int, const Stub*, Tracklet*);
+    std::vector<double> getInventedCoordsExtended(unsigned int, const Stub*, Tracklet*);
+    std::vector<const Stub*> seedStubCoordsFromTracklet(unsigned int, Tracklet*, std::vector<const Stub*>);
```

Comment:
Add comment explaining what seedStubCoordsFromTracklet does. Also the name of this function is misleading. It doesn't return the coordinates. It returns the invented stubs.

Reply:
done, and name changed